Makefile

```diff
@@ -22,8 +23,19 @@ vm-sanity-checks:
 platform-sanity-checks:
 	cd platform_sanity_checks && ${MAKE} ${PASS_DOWN_ARGS}
 
+rmsr/rmsr.ko:
+ifeq ($(shell uname -s),Linux)
+	# Allow git to fail if rmsr has already been cloned.
+	${GIT} clone https://github.com/softdevteam/rmsr.git | true
```

Comment:
I disagree. rmsr is a dependency of krun, not of the warmup experiment. If we want rmsr to be external, then we can re-instate a config variable that allows the user to  specify where the module is, then build.sh in the warmup_experiment can build it, but I don't really see any need.